app/src/modules/users/routes/item.vue

```diff
@@ -102,7 +102,7 @@
 			<div v-if="isNew === false" class="user-box">
 				<div class="avatar">
 					<v-skeleton-loader v-if="loading || previewLoading" />
-					<img v-else-if="avatarSrc" :src="avatarSrc" :alt="item.email" />
+					<img v-else-if="avatarSrc" :src="avatarSrc" :alt="Avatar" />
```

Comment:
Should probably be localized

Reply:
Also, this will crash as it tries to read `Avatar` as a variable. 

```suggestion
					<img v-else-if="avatarSrc" :src="avatarSrc" alt="Avatar" />
```

or 

```suggestion
					<img v-else-if="avatarSrc" :src="avatarSrc" :alt="t('avatar')" />
```